spec/game_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'game'
+
+RSpec.describe Game do
+  before :each do
+    @game = Game.new
+  end
+
+  it 'prompts user for a move and returns move as integer' do
```

Comment:
I'd prefer to read: "prompts the user for their move and and returns the move as an integer"

Reply:
Thanks, that is much clearer. I'll change it